<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>XIVN1987</author>
 <class>SERCOM</class>
 <widget class="QWidget" name="SERCOM">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>466</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>XIVN1987 SERCOM</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Image/serial.ico</normaloff>Image/serial.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="vLayout0">
   <item>
    <widget class="QTextEdit" name="txtMain">
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QwtPlot" name="qwtPlot"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hLayout1">
     <item>
      <widget class="QLabel" name="lblDLL">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>JLinkARM.dll路径：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="linDLL"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnDLL">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hLayout2">
     <item>
      <widget class="QLabel" name="lblMap">
       <property name="text">
        <string>项目.map文件路径：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="linMap"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnMap">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hLayout3">
     <item>
      <widget class="QPushButton" name="btnClear">
       <property name="text">
        <string>清除</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="hSpacer31">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="lblStat">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>Image/inclosing.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnOpen">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>104</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>打开连接</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QComboBox" name="cmbMode">
       <item>
        <property name="text">
         <string>文本</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>波形</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
